browser-test/src/support/admin_programs.ts

```diff
@@ -58,6 +58,16 @@ export interface BlockSpec {
   questions: QuestionSpec[]
 }
 
+export interface ProgramSelectorSpec {
+  name: string
+  isDisabled?: boolean
+}
+
+export interface ProgramBlockSpec {
+  programSelector: ProgramSelectorSpec
```

Comment:
(thought) Maybe we don't make this an object and just put the two properties of `ProgramSelectorSpec` onto `ProgramBlockSpec`

Reply:
Yeah I considered this... I think the selector is more extensible and the naming makes it more clear that you are selecting rather than creating/defining (though apparently not that clear since you didn't find it so :|). It also avoids the need for /* isProgramDisabled = */ which authors might skip.

On the other side, it doesn't seem that likely that we'll need another selector criteria and if it's only clearer to me and prompts confusion with other selector concepts, then I'm fine to inline it.

What do you think?

Reply:
As a side thought, this selector spec pattern would be reusable/extendible for methods like goToEditBlockVisibilityPredicatePage that currently take a program name but could automatically support disabled programs if they used ProgramSelectorSpec (or a ProgramBlockSelectorSpec that extends it with also a block name).

Reply:
I see where you're coming from and it's not a bad idea, but I don't know if we'll ever really need to be that extensible. Let's inline it for now and refactor it down the road if we find a more pressing need.

Reply:
It looks like I mis-interpreted your suggestion, or maybe I forgot what I was thinking yesterday, but anyway I don't like putting programName and isProgramDisabled into a "Spec" class because they are not part of a spec (i.e. entity definition which all the specs currently are) they're for finding the program. So I just changed the PR to use BlockSpec instead of QuestionSpec.